llvm/test/Transforms/JumpThreading/thread-debug-info.ll

```diff
@@ -91,6 +92,47 @@ exit:                                             ; preds = %bb.f4, %bb.f3, %bb.
   ret void, !dbg !29
 }
 
+; Test for the cloning of dbg.values on elided instructions -- down one path
+; being threaded, the and in the function below is optimised away, but it's
```

Comment:
Could you demark "and" so it doesn't look like a word, e.g. \`and\`?

nit: it's -> its (apostrophe is possessive unless there's already a contraction, I think is the rule. At the very least, "it's" is always "it is" and never "belonging to it").

There's another "it's" in the next sentence that wants the same treatment.